mealoca/me-aloca-backend
# Feature/crud laboratory

#### What is the purpose of this pull request?

Create the laboratory model, router and controller. This branch will be used to integrate this model with the others.

#### What problem is this solving?

We needed a Laboratory CRUD.

#### How should this be manually tested?

Using postman to make requests for the server and see if the api is correctly responding and modifying the resources.

#### Screenshots or example usage

#### Types of changes

- [x ] New feature (a non-breaking change which adds functionality)
